ext/standard/streamsfuncs.c

```diff
@@ -432,7 +432,7 @@ PHP_FUNCTION(stream_get_contents)
 	if (maxlen_is_null) {
 		maxlen = (ssize_t) PHP_STREAM_COPY_ALL;
 	} else if (maxlen < 0 && maxlen != PHP_STREAM_COPY_ALL) {
-		php_error_docref(NULL, E_WARNING, "Length must be greater than or equal to 0, or -1");
+		php_error_docref(NULL, E_WARNING, "Argument #2 ($maxlength) must be greater than or equal to 0, or -1");
```

Comment:
```suggestion
		php_error_docref(NULL, E_WARNING, "Argument #2 ($maxlength) must be greater than or equal to -1");
```
And can't this also be a ValueError?

Reply:
Ah, yes, I also found this wording weird, but forgot to ask it! As the warning has been added by Nikita recently, my guess is that it's not an exception for a reason. (?). But yeah, asking him is the best idea :)